### DISCUSSION TITLE
Integration examples for Astro and Remix?

### DISCUSSION THREAD

**Discussion by JoshuaKGoldberg** [vanilla-extract-css/vanilla-extract#1266] [Ideas]
I want to use Vanilla Extract with some  sites on not-Next.js: Astro, Remix, and potentially others. Although there technically is support for...

* Astro: per #678
* Remix: per #178 -> https://github.com/remix-run/remix/pull/5040

...https://github.com/vanilla-extract-css/vanilla-extract/tree/master/examples doesn't have examples for either of them. Would the project be open to keeping examples maintained for popular React meta-frameworks other than Next.js?

Bonus points if we can also have equivalents of https://vanilla-extract.style/documentation/integrations/next too!

I'd be more than happy to send PRs to get these in myself.

_edit: I forgot to check the 'I have done a search for similar discussions' checkbox - but it let me post anyway? interesting..._

Edit 2: courtesy of @brophdawg11, Remix itself has a Vanilla Extract guide: https://remix.run/docs/en/main/styling/vanilla-extract. But I'd still like to request _something_ -at the very list a link to that guide- be put in the Vanilla Extract site. I'd missed that guide at first!

**Comment by askoufis** [comment]
PRs would absolutely be welcome :)

**Reply by JoshuaKGoldberg** [reply]
Awesome, thanks! I'll send them.

@askoufis just to be clear so I know what to send: what would you prefer for Remix? A full example, a link to the existing Remix site, ...?

**Reply by askoufis** [reply]
Probably just a link to the remix site, as well as a link to the vite integration since remix is now vite-based.

**Reply by JoshuaKGoldberg** [reply]
Sent!
* #1281
* #1283
